benchmarks/src/garage_benchmarks/experiments/algos/ddpg_garage_tf.py

```diff
@@ -42,7 +42,7 @@ def ddpg_garage_tf(ctxt, env_id, seed):
     deterministic.set_seed(seed)
 
     with LocalTFRunner(ctxt) as runner:
-        env = GarageEnv(normalize(gym.make(env_id)))
+        env = normalize(GymEnv(gym.make(env_id)))
```

Comment:
```suggestion
        env = normalize(GymEnv(env_id))
```

The garage codebase is the greatest source of usage examples, so it should always strive to use the API idiomatically. In this case, the string option is here to save people to repetition of calling `gym.make`. If passing a string is an option, we should use it.

Please update this usage throughout examples/tests/docs where applicable.